### PR TITLE
fix: no zoom buttons on mobile

### PR DESCRIPTION
**Pull request by kommander**
what the title says

### CODE REVIEW

File: src/components/CovMap.tsx [hunk 1]
```diff
@@ -75,6 +76,7 @@ export const CovMap = () => {
   const dispatch = useThunkDispatch();
   const urlParams = useParams<{ subPage?: string }>();
   const history = useHistory();
+  const isMobile = useMediaQuery("(max-width:600px)");
```
Comment (DanielHabenicht) [inline src/components/CovMap.tsx:79]: just one remark can we centralize the break-points somewhere?

**Review by DanielHabenicht — APPROVED** [review]

File: src/components/CovMap.tsx (hunk 1 above)
Comment (kommander) [reply src/components/CovMap.tsx:79]: Yes, was thinking about that as well... would like to keep it as a reminder for #281

**Comment by DanielHabenicht** [comment]
@all-contributers add @sheraben for code

**Comment by DanielHabenicht** [comment]
sry for taking over this pr to try it out

**Pull request merged**